### PR TITLE
Created first draft of CODEOWNERS file

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+
+# Prefer GitHub teams over individuals to make it easier to find a suitable owner
+# when someone is OOO or leaves the team.
+
+# terraform-samples-owners is in charge of infrastructure in this repository ##### TODO: create this GitHub team #####
+# terraform-samples-reviewers reviews Terraform sample code for adherence to sample guidelines
+*                                      @GoogleCloudPlatform/terraform-samples-owners @GoogleCloudPlatform/terraform-samples-reviewers
```
Comment (rogerthatdev) [inline .github/CODEOWNERS:13]: I don't know if we can add teams from other orgs to access repos in this org. Keep this PR in draft until we verify, or until we create the corresponding terraform-google-modules teams

**Review by rogerthatdev — COMMENTED** [review]
I don't know if we can add teams from other orgs to access repos in this org. Keep this PR in draft until we verify, or until we create the corresponding terraform-google-modules teams

**Pull request closed without merging**